gammapy/modeling/tests/test_fit.py

```diff
@@ -249,9 +250,9 @@ def test_stat_surface_reoptimize():
     ]
 
     assert_allclose(list(result["stat_scan"]), expected_stat, atol=1e-7)
-    assert_allclose(
-        result["fit_results"][0][0].total_stat, result["stat_scan"][0][0], atol=1e-7
-    )
+    # assert_allclose(
+    #     result["fit_results"][0][0].total_stat, result["stat_scan"][0][0], atol=1e-7
+    # )
```

Comment:
is this failing ?

Reply:
The object result["fit_results"] contains now the status of the fit (a Boolean), and not the full object 'result = self.optimize'